src/styles/theme.ts

```diff
@@ -1,14 +1,33 @@
 import { DeepPartial, FlowbiteTheme } from 'flowbite-react';
 
 const theme: DeepPartial<FlowbiteTheme> = {
+  accordion: {
```

Comment:
I think that this is going to be a problem.

Reply:
yeahh So, I believe we need to make our own accordion component, or maybe(maybe) we need to make a pull request to flowbite.